plugins/formal-verification/testData/diagnostics/verifies/stdlib_replacement_tests.fir.diag.txt

```diff
@@ -65,7 +67,9 @@ method f$useRuns$TF$T$Int(p$x: Ref) returns (ret$0: Ref)
   inhale df$rt$isSubtype(df$rt$typeOf(ret$0), df$rt$unitType())
 }
 
-/stdlib_replacement_tests.kt:(403,410): info: Generated Viper text for useAlso:
+/stdlib_replacement_tests.kt:(415,422): info: Generated Viper text for useAlso:
+field bf$size: Ref
```

Comment:
I'm curious, why does this field show up in so many more files after this change?

Reply:
Oh, it's because of the array in the varargs, I take it.